docusaurus.config.ts

```diff
@@ -147,10 +147,10 @@ const config: Config = {
 
   customFields: {
     /** Use Testuser in local dev: set TEST_USERNAME to the test users email adress*/
-    TEST_USERNAMES: process.env.TEST_USERNAMES?.split(';') || [],
+    TEST_USERNAMES: (process.env.TEST_USERNAMES?.split(';') || []).map((u) => u.trim()).filter(u => !!u),
```

Comment:
we could sanitize the usernames in case any spaces were added or empty values were used